src/store/auth.module.js

```diff
@@ -10,10 +11,10 @@ const state = {
 }
 
 const getters = {
-  getUser (state) {
+  [GET_CURRENT_USER] (state) {
```

Comment:
It's fine for the moment, but I have a slight reservation for requiring getters type.
Is there a particular reason to prefer this pattern? It seems that it adds a dependency to the component file.
I would love to have your thoughts about this choice.

Reply:
I'm used to add getters to the vuex store. I don't know if thats a best practice, just think it's nice to have one place for all posible getters on the state. There is discussion whether it is a best practice.